src/services/db/GitHubService.js

```diff
@@ -487,6 +490,35 @@ class GitHubService {
       throw err
     }
   }
+
+  async changeRepoPrivacy(sessionData, shouldMakePrivate) {
+    const { siteName, isomerUserId } = sessionData
+    const endpoint = `${siteName}`
```

Comment:
sanity check here, wouldnt endpoint refer to `repos/<repoName>` and `siteName` refer to just `repoName`? should we just use the variable `siteName` instead?

Reply:
This is referring to the endpoint still! The actual endpoint to call is `/repos/<orgname>/<repoName>`, but our axios instance prefixes with `/repos/<orgname>/`, so the remaining endpoint is just `<repoName>` (referred to as `siteName` here